reana_db/models.py

```diff
@@ -252,6 +258,20 @@ def has_exceeded_quota(self):
             r.quota_limit != 0 and r.quota_used >= r.quota_limit for r in self.resources
         )
 
+    def get_workflow_overload_priority(self):
+        """Get priority factor based on the number of current workflows ``running``."""
+        max_concurrent_workflows = REANA_MAX_CONCURRENT_BATCH_WORKFLOWS
+        running_count = self.workflows.filter(
```

Comment:
Question: shouldn't we filter workflows by user? If I'm not mistaken now if user A submits 10 workflows and after user B submits his first workflow, the priority of that workflow will still be penalized?

Reply:
`self.workflows` is a 1:n (User:Workflow) relationship that already filters the workflows whose owner is `self`.

Reply:
I had a suspicion that this is the case, thanks for clarifying!